linux/network_device_ops/update_systemd_network_config.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2025 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Update systemd network config file for primary or secondary network adapter
+#
+- name: "Set fact of systemd network config template on {{ vm_guest_os_distribution }}"
+  ansible.builtin.set_fact:
+    systemd_network_config_template: "systemd_network_conf.j2"
+
+- name: "Get systemd network config file for '{{ network_adapter_name }}'"
```

Comment:
I saw all these 3 blocks have included this task file, so it can be included once at above.